libs/ui-lib/lib/common/components/clusterConfiguration/ControlPlaneNodesDropdown.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import { Dropdown, DropdownItem, DropdownToggle, FormGroup, Tooltip } from '@patternfly/react-core';
+import { useTranslation } from '../../hooks/use-translation-wrapper';
+import { getFieldId, StaticField } from '../..';
+import { useField } from 'formik';
+
+interface ControlPlaneNodesOption {
+  value: string;
+  label: string;
+}
+
+const isDropdownItemEnabled = (controlPlaneNodeCount: string, isNutanix?: boolean): boolean => {
+  return (controlPlaneNodeCount === '1' && !isNutanix) || controlPlaneNodeCount !== '1';
+};
+
+const ControlPlaneNodesDropdown = ({
+  isDisabled = false,
+  isNutanix,
+}: {
+  isDisabled?: boolean;
+  isNutanix?: boolean;
+}) => {
+  const { t } = useTranslation();
+  const [{ name, value }, , { setValue }] = useField<string | ''>('controlPlaneAgents');
+  const [current, setCurrent] = React.useState<string>('3');
+
+  const options: ControlPlaneNodesOption[] = [
+    { value: '1', label: t('ai:1 (Single Node OpenShift - not highly available cluster)') },
+    { value: '3', label: t('ai:3 (highly available cluster)') },
+    { value: '4', label: t('ai:4 (highly available cluster+)') },
+    { value: '5', label: t('ai:5 (highly available cluster++)') },
```

Comment:
Do we need any checks for OCP version or/and CPU arch, maybe something else to enable these new options?

Reply:
@danmanor for CIM, do we need the same validations that for SaSS (OCP >=4.18, cpu_arch=x86 and platform=none o baremetal)?

Reply:
but probably best to rely on assisted-service - querying the feature support endpoint with version, arch, platform

Reply:
we need to add manual conditions for this